### DISCUSSION TITLE
Mashify middleware is now available as a gem

### DISCUSSION THREAD

**Discussion by graaff** [lostisland/awesome-faraday#24] [General]
It's still listed as part of the old faraday-middleware, but there is now a faraday 2 compatible gem for it:
https://rubygems.org/gems/faraday-mashify
https://github.com/sue445/faraday-mashify

**Comment by iMacTia** [comment]
Thank you for pointing this out @graaff 🙏 
@olleolleolle there seems to have been a small copy-paste error in [your recent commit](https://github.com/lostisland/awesome-faraday/commit/77fee5c8ce2e44b4a1d5f2875d7330685ec019db)?

**Comment by olleolleolle** [comment]
Ah, I may have confused someone - in order not confuse, I removed the link to the now-fixed-for-2.0 middleware that was in faraday_middleware. By removing a link, I made the document slightly clearer.

Hope this helps! 

(Closing!)
